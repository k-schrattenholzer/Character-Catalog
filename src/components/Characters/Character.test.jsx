import { render } from '@testing-library/react';
import Character from './Character.jsx';

it('should render a character detail', () => {
  const mockChar = {
    image: 'http://example.com/image.jpg',
    name: 'Katie',
    species: 'Unknown',
    status: 'Unknown'
  }
  const { container } = render (
    <Character character={mockChar}/>
  )

  expect(container).toMatchSnapshot();
})