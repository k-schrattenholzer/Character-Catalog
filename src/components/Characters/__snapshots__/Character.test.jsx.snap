// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a character detail 1`] = `
<div>
  <figure>
    <img
      alt="Katie"
      src="http://example.com/image.jpg"
    />
    <figcaption>
      <h3>
        Katie
      </h3>
      <p>
        Unknown
      </p>
      <p>
        Status: 
        Unknown
      </p>
    </figcaption>
  </figure>
</div>
`;
